### PR TITLE
More Random Stuff™️

### PR DESCRIPTION
**Pull request by halotroop2288**
Just started working on networking stuff. There's some other random garbage in there too. Even some stuff I thought I already submitted...

80.47% of classes are now mapped.
61.92% of methods, 55.70% of fields and 36.59% of parameters are also mapped.

## Non-mappings changes:
- Updated Enigma, and consequently Java and Gradle to their latest versions.
- Switched to Quilt Loader 0.13.0 from Fabric Loader 0.11.6
- Switched to Quilt's Stitch 0.6.2 from Fabric's Stitch 0.6.1
- Updated tiny-remapper to version 0.4.3 from 3.1.72
- Implemented proper check for Github Actions vs local build.
	- Local builds will now be versioned as `1.6.4+build.local` while Github Actions builds will be versioned with the current date and time, as usual for Legacy Fabric.

### CODE REVIEW

File: mappings/net/minecraft/client/ClientBrandRetriever.mapping [hunk 1]
```diff
@@ -1,2 +1,2 @@
 CLASS net/minecraft/client/ClientBrandRetriever net/minecraft/client/ClientBrandRetriever
-	METHOD getClientModName ()Ljava/lang/String;
```
Comment (halotroop2288) [inline mappings/net/minecraft/client/ClientBrandRetriever.mapping:2]: Oh god this looks like another one of those changes that will cause the game to crash. TBD

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-enigma_version=0.27.3
+enigma_version=1.3.2
```
Comment (Grayray75) [inline gradle.properties:4]:
```suggestion
enigma_version=1.0.0
```
Version 1.0.0 is the last one with supports Java below 16

Comment (halotroop2288) [reply gradle.properties:4]: Doesn't matter for Yarn. None of its dependencies are (should be) inherited. It's purely for mappings. Using The latest version means I get to use all the newest features.

Comment (halotroop2288) [reply gradle.properties:4]: Oh, *sigh*. Right. That means that the publish workflow will have to use JDK16, which will generate incompatible bytecode...

Comment (halotroop2288) [reply gradle.properties:4]: As long as we're not using Yarn for patches, that shouldn't be a problem though...

Comment (halotroop2288) [reply gradle.properties:4]: Hmmm, it looks like something else is causing a problem. I tried reverting Enigma, Gradle, and JDK locally, but build is still failing at [build.gradle line 483](https://github.com/Better-Than-Updates-MC/Mappings/blob/pr/1.6.4/update/build.gradle#L483)

Comment (Grayray75) [reply gradle.properties:4]: for me too... are the intermediary broken??

Comment (halotroop2288) [reply gradle.properties:4]: Unlikely. Everything was working up to this last couple of commits.

**Comment by halotroop2288** [comment]
I'm pretty happy with the way things are going.
current progress

**Comment by halotroop2288** [comment]
Hmm actually I will do more later. Might get it up to 80% classes or more before it's ready.

File: mappings/net/minecraft/structure/feature/ChunkPosHashProivder.mapping [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_2229 net/minecraft/structure/feature/ChunkPosHashProivder
+	FIELD field_9699 chunkX I
+	FIELD field_9700 chunkY I
```
Comment (halotroop2288) [inline mappings/net/minecraft/structure/feature/ChunkPosHashProivder.mapping:3]: *chunkZ

Chunks don't have Y positions.

Note to self: Fix all of these.

**Review by Grayray75 — APPROVED** [review]

**Review by hYdos — APPROVED** [review]
lgtm

**Pull request merged**